lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
> Minor: what about renaming `payment_partial_name` into `payment_type`? 
> <snipped>
> Not sure about talking about partials in the config file?

@pmanrubia: I agree that mentioning partials in the data isn't ideal. Given that I can't see an easy way round it at the moment I prefer to make it explicit that the data represents the name of a partial to render. Using something like `payment_type` might lead someone to think they can store an arbitrary string against that key.